### ISSUE TITLE
Error when downloading "Boku no Kokoro no Yabai Yatsu Season 2" from animepahe

### ISSUE THREAD

**Issue by hillz2** [SenZmaKi/Senpwai#27]
I got this error when trying to download that anime:
```
19:00:24 root@2b54cb835a09 ~ senpcli -sd sub -s pahe -se 1 -ee 50 -q 1080p -msd 10 "Boku no Kokoro no Yabai Yatsu Season 2"

                                   .__  .__
  ______ ____   ____ ______   ____ |  | |__|
 /  ___// __ \ /    \\____ \_/ ___\|  | |  |
 \___ \\  ___/|   |  \  |_> >  \___|  |_|  |
/____  >\___  >___|  /   __/ \___  >____/__|
     \/     \/     \/|__|        \/

Setting end episode to 9 since the anime only has 9 episodes
Downloading to: /Media/Anime/Boku no Kokoro no Yabai Yatsu Season 2
Traceback (most recent call last):
  File "/usr/local/bin/senpcli", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/senpwai/senpcli/main.py", line 700, in main
    initiate_download_pipeline(parsed, *anime_and_anime_details)
  File "/usr/local/lib/python3.11/dist-packages/senpwai/senpcli/main.py", line 665, in initiate_download_pipeline
    handle_pahe(parsed, anime, anime_details)
  File "/usr/local/lib/python3.11/dist-packages/senpwai/senpcli/main.py", line 510, in handle_pahe
    if already_has_all_episodes(
       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/senpwai/senpcli/main.py", line 461, in already_has_all_episodes
    lacked_eps_page_links = lacked_episodes(
                            ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/senpwai/utils/scraper.py", line 370, in lacked_episodes
    return [
           ^
  File "/usr/local/lib/python3.11/dist-packages/senpwai/utils/scraper.py", line 371, in <listcomp>
    episode_page_links[eps_number - first_eps_number]
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
IndexError: list index out of range
```

Can you fix that ? switching to `gogo` doesn't help

**Comment by SenZmaKi** [comment]
I'll fix it in the next update, for now use Gogoanime cause I just released an update today that fixes it.

Also it's preferred if you report issues on the [discord](https://discord.gg/invite/e9UxkuyDX2) for easier tracking and faster responses.

**Comment by hillz2** [comment]
Noted thanks

**Comment by SenZmaKi** [comment]
Fixed #30
